### PR TITLE
Add docs on how to training/inference with flair text/token-classification model

### PR DESCRIPTION
**Pull request by sakares**
- [x] complete how to train text-classification docs
 - [x] complete how to train token-classification docs

### CODE REVIEW

File: docs/guides/cookbook.ipynb [hunk 1]
```diff
@@ -554,6 +795,212 @@
     "rb.log(records=record, name=\"flair-pos-tagging\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Training\n",
+    "\n",
+    "Let's read a Rubrix dataset, prepare a training set, save to `.ะปะ` for loading with flair `ColumnCorpus` and train with flair `SequenceTagger` "
```
Comment (dvsrepo) [inline docs/guides/cookbook.ipynb]: I guess this supposed to be `.txt`

File: docs/guides/cookbook.ipynb [hunk 2]
```diff
@@ -554,6 +795,212 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# 2. Pre-processing to BIO scheme before saving as .txt file\n",
+    "\n",
+    "# Use origin prediction as label for reproduce purpose, you need to use your own label\n",
```
Comment (dvsrepo) [inline docs/guides/cookbook.ipynb]:
```suggestion
    "# Use original predictions as annotations for demonstration purposes, in a real use case you would use the `annotations` instead\n",
```

**Review by dvsrepo — CHANGES_REQUESTED** [review]
Very good job @sakares ! Just two minor change requests

File: docs/guides/cookbook.ipynb (hunk 1 above)
Comment (sakares) [reply docs/guides/cookbook.ipynb]: Alright, fixed typo

File: docs/guides/cookbook.ipynb (hunk 2 above)
Comment (sakares) [reply docs/guides/cookbook.ipynb]: Changed comment upon the suggestion

**Review by dvsrepo — APPROVED** [review]

**Comment by dcfidalgo** [comment]
@sakares Thanks a lot for your contribution! Could you maybe rebase this branch onto our master branch? Then I think this would be good to merge.

**Comment by sakares** [comment]
Hi! I am not sure if I rebase this branch with rubrix master branch correctly. I feel like I mess with something.
Pardon me for troubling around 😅

**Comment by dcfidalgo** [comment]
Ok, I think I cleaned up a bit :) Could you quickly check if all your changes are still there?

**Comment by sakares** [comment]
Yes, all changes are still there. Look good to me.

**Comment by sakares** [comment]
Hi, please feel free to add the label `hacktoberfest-accepted` if this PR passes qualification. Thanks!

**Pull request merged**